src/gennylib/benchmark/orchestrator_benchmark.cpp

```diff
@@ -1,101 +0,0 @@
-// Copyright 2019-present MongoDB Inc.
```

Comment:
💥 👍 
i think we can also remove `PhaseLoop_benchmark.cpp`?

Reply:
I was planning on leaving it in there because it does multi-threaded runs, which this benchmark doesn't do. It's also a very stable test and we haven't needed to update the threshold ever.